Tests/CreateAPITests/Compliling/PackageManifest.swift

```diff
@@ -45,6 +45,26 @@ struct PackageManifest {
             return ".byName(name: \(dependency.name.inDoubleQuotes), condition: .when(platforms: [.macOS]))"
         }
     }
+
+    static func productName(in packageURL: URL) throws -> String? {
+        let manifestURL = packageURL.appendingPathComponent("Package.swift")
+        guard FileManager.default.fileExists(atPath: manifestURL.path) else { return nil }
+
+        let contents = try String(contentsOf: manifestURL)
+        let range = NSRange(contents.startIndex ..< contents.endIndex, in: contents)
+        let regex = try NSRegularExpression(pattern: #"library\(name: "(.*)", targets"#)
+
+        guard let match = regex.firstMatch(in: contents, range: range) else {
+            return nil
+        }
+
+        let groupRange = match.range(at: 1)
+        guard let range = Range(groupRange, in: contents) else {
+            return nil
+        }
+
+        return String(contents[range])
+    }
```

Comment:
Previously when running snapshot tests we could just read the `package` option from the parsed `Generate` command but that is no longer the case because the config file (or overrides) aren't loaded until after running the generator and there is no way to pass that information back out.

Instead, I opted to read the package name from the generated **Package.swift** file if it exists. 

There might be a better approach for this in the future but it should do the trick for now.

Reply:
The folder name: `Compliling` -> `Compiling`